src/bvals/neighbor_block.hpp

```diff
@@ -144,9 +146,6 @@ struct NeighborBlock {
   block_ownership_t ownership;
   RegionSize block_size;
 
-  void SetNeighbor(LogicalLocation inloc, int irank, int ilevel, int igid, int ilid,
-                   int iox1, int iox2, int iox3, NeighborConnect itype, int ibid,
-                   int itargetid, int ifi1 = 0, int ifi2 = 0);
```

Comment:
This is no longer used anywhere (constructors are used instead), so I removed it.